### PR TITLE
Handle updates for published content

### PR DESCRIPTION
**Pull request by amagdas**
Things to be included in this PR:
- [x] add sign_off field to user creation and authoring screen
- [x] filter list actions for published content
- [x] handle Kill action for published content
- [x] add more metadata fields to the authoring screen
- [x] filter for published items, show all published versions
- [x] support for scheduled publishing [WIP] https://github.com/superdesk/superdesk/pull/492
- [x] remove autosave versions when publishing 
- [x] prevent reverting versions after a story is published

### CODE REVIEW

File: client/app/scripts/superdesk-archive/module.js [hunk 1]
```diff
@@ -172,31 +172,14 @@ define([
...
-                                $rootScope.$broadcast('item:duplicate');
-                            }, function(response) {
-                                data.item.error = response;
-                            })
-                        ['finally'](function() {
-                            data.item.actioning.archiveContent = false;
-                        });
-                    }],
-                    */
+                    icon: 'archive',
```
Comment (sivakuna-aap) [inline client/app/scripts/superdesk-archive/module.js]: Why are we replacing copy icon with archive icon here? In personal desk the requirement was to display copy icon.

Comment (amagdas) [inline client/app/scripts/superdesk-archive/module.js]: Merge blunder, will fix :smiley:

**Comment by marwoodandrew** [comment]
:+1:

**Comment by sivakuna-aap** [comment]
:+1:

**Pull request merged**